packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -3384,7 +3384,190 @@ curl -v -X PUT \
+{
+  "id": "0oa1ivcatpQDkMraA1d7",
+  "name": "oidc_client",
+  "label": "oauth2 client app 1",
+  "status": "ACTIVE",
+  "lastUpdated": "2021-10-04T22:48:42.000Z",
+  "created": "2021-10-04T22:42:38.000Z",
+  "accessibility": {
+    "selfService": false,
+    "errorRedirectUrl": null,
+    "loginRedirectUrl": null
+  },
+  "visibility": {
+    "autoLaunch": false,
+    "autoSubmitToolbar": false,
+    "hide": {
+      "iOS": true,
+      "web": true
+    },
+    "appLinks": {
+      "oidc_client_link": true
+    }
+  },
+  "features": [],
+  "signOnMode": "OPENID_CONNECT",
+  "credentials": {
+    "userNameTemplate": {
+      "template": "${source.login}",
+      "type": "BUILT_IN"
+    },
+    "signing": {
+      "kid": "Hn7zyyJ1XhKhAHa6MaNC2GjWmw3F5Pa_TxGAx9ojch0"
+    },
+    "oauthClient": {
+      "autoKeyRotation": true,
+      "client_id": "0oa1ivcatpQDkMraA1d7",
+      "client_secret": "LD1G1-qeKo_KfM3VQx3Ass8aaGlYxuCq_cqCGlKF",
+      "token_endpoint_auth_method": "client_secret_post"
+    }
+  },
+  "settings": {
+    "app": {},
+    "notifications": {
+      "vpn": {
+        "network": {
+          "connection": "DISABLED"
+        },
+        "message": null,
+        "helpUrl": null
+      }
+    },
+    "notes": {
+      "admin": null,
+      "enduser": null
+    },
+    "oauthClient": {
+      "client_uri": "http://localhost:8080",
+      "logo_uri": "http://developer.okta.com/assets/images/logo-new.png",
+      "redirect_uris": [
+        "https://example.com/oauth2/callback",
+        "myapp://callback"
+      ],
+      "response_types": [
+        "token",
+        "id_token",
+        "code"
+      ],
+      "grant_types": [
+        "implicit",
+        "authorization_code"
+      ],
+      "application_type": "native",
+      "issuer_mode": "ORG_URL",
+      "idp_initiated_login": {
+        "mode": "DISABLED",
+        "default_scope": []
+      }
+    }
+  },
+  "profile": {
+    "label": "oauth2 client app 1"
+  },
+  "_links": {
+    "uploadLogo": {
+      "href": "https://dev-adatla-op3.oktapreview.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/logo",
+      "hints": {
+        "allow": [
+          "POST"
+        ]
+      }
+    },
+    "appLinks": [
+      {
+        "name": "oidc_client_link",
+        "href": "https://dev-adatla-op3.oktapreview.com/home/oidc_client/0oa1ivcatpQDkMraA1d7/aln177a159h7Zf52X0g8",
+        "type": "text/html"
+      }
+    ],
+    "groups": {
+      "href": "https://dev-adatla-op3.oktapreview.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/groups"
+    },
+    "logo": [
+      {
+        "name": "medium",
+        "href": "https://op3static.oktacdn.com/assets/img/logos/default.6770228fb0dab49a1695ef440a5279bb.png",
+        "type": "image/png"
+      }
+    ],
+    "users": {
+      "href": "https://dev-adatla-op3.oktapreview.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/users"
+    },
+    "deactivate": {
+      "href": "https://dev-adatla-op3.oktapreview.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/lifecycle/deactivate"
```

Comment:
```suggestion
      "href": "https://example.okta.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/lifecycle/deactivate"
```

Reply:
@susanharper-okta modified urls to generic example.okta domain.